### DISCUSSION TITLE
NOUS A1T with Tasmota dead

### DISCUSSION THREAD

**Discussion by m4ding** [arendst/Tasmota#17594] [Q&A]
Hi,

I bought a set of four Nous A1T with tasmota preflashed. Unfortunately, one of them is dead meaning that it doesnt appear in the wifi with tasmota-xyz and the button is also not lightning up when it is pushed (and no light when the device is put into the socket. Neither pushing the button for 40 seconds nor performing a Fast Power Cycle Device Recovery (https://tasmota.github.io/docs/Device-Recovery/#fast-power-cycle-device-recovery) works. Basically, I dont know if turning the device on and of six times quickly works since I dont hear the click sound of the device when activating power. Before I send all four back, I just want to make sure that I tried everything possible.

Thanks
mading

**Comment by meganom** [comment]
Hi mading,
It was not working from the beginning?
I found some related information but for A1, not A1T version: https://github.com/arendst/Tasmota/discussions/12921

In my case i have updated the original FW to KNX version over OTA. It was working perfectly one day and later, when I tried to update KNX configuration device died and doesn't show any reaction. I also tried to follow all the recovery steps, but no success. 
So, I am going to open it and reflash. 

Update: another discussion about A1T: https://github.com/arendst/Tasmota/discussions/16244

Regards
Sergey
